scripts/processing/01-anatomy.py

```diff
@@ -74,7 +74,7 @@ def process_subject_anat(subject_id, force_recon_all=False):
             shutil.copy(f_src, f_dst)
 
     # Fix the headers for subject 19
-    if not op.isfile(op.join(dst_flash, "mef05_0.nii.gz")):
+    if subject_id == 19:
```

Comment:
This will make it so this gets re-run every time you run the script, which is not optimal

Reply:
What was there before was not working and this cheap to run anyway, no?

Reply:
It's not a lot of overhead, true, but it would have been better to fix the check rather than re-run it. We can leave it though

Reply:
I guess the problem is not missing flash file, but rather a corrupted header. So, you'd have to check the header to know if it was broken or not. But yeah, let's leave it I'd say